index.html

```diff
@@ -78,11 +94,13 @@ <h3 class="card-main-title">Profesional Art Printing Data</h3>
                             </ul>
                         </section>  
                         </section> 
+                        <img id="cursor" src="images/Shape.png">
                         <button class="card-big-btn" >See project</button>    
                     </section>
-                    <section class="post">
+                    <section class="post post2">
                         <section class="card-contents">
                             <h3 class="card-main-title">Profesional Art Printing Data</h3>
+                            <h3 class="card-main-title1">Data Dashboard Healthcare </h3>
                             <p class="card-description">A daily selection of privately personalized reads;
                                no accounts or sign-ups required. Has been the industry's standard.
                             </p>
```

Comment:
- [ ] As I mentioned above, the content of this section also does not respond well between the screen sizes, please look into this too. 👋 
- Screenshot:
![Screenshot 2023-04-01 094526](https://user-images.githubusercontent.com/107632780/229277085-033015c0-f12d-4246-b3f3-ca0d4b9c7d51.png)